### PR TITLE
fixes admin circle selector

### PR DESCRIPTION
**Pull request by f1lander**
image
Fixes:

- red line remove
- admin circle selector in mobile

### CODE REVIEW

File: src/components/CirclesHeaderSection/CirclesHeaderSection.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { makeStyles, Theme } from '@material-ui/core';
 
 import { CircleButton } from 'components';
 import { useApiBase } from 'hooks';
-import { useSelectedCircle, useCircles } from 'recoilState';
+import { useSelectedCircle, useCircles } from 'recoilState/app';
```
Comment (exrhizo) [inline src/components/CirclesHeaderSection/CirclesHeaderSection.tsx:9]: Undo this, I should probably delete index, @levity asked me to import from the individual files

**Review by exrhizo — COMMENTED** [review]
This and moving the sections, like in the chat
Otherwise, Looks Good To Me, LGTM

**Comment by levity** [comment]
I get this when I hit the vercel preview app in mobile safari:

image

**Comment by f1lander** [comment]
Mmm I already fixed, let check if I forgot to pushed the fix

**Comment by levity** [comment]
OK, that's fixed now. There's still something weird going on with scrolling though. When I open the menu, I can scroll down so that the Admin View and Circle Selector text come into view, but as soon as I let go of the screen, the scrolling "snaps back" so that they are now out of view again, so I can't tap them. This is on Mobile Safari on iPhone 12.

**Comment by levity** [comment]
Here's a screen recording to demonstrate 

https://user-images.githubusercontent.com/193667/144670814-bbb5017b-fbea-47ba-9ddd-315e2a715901.MOV

**Comment by f1lander** [comment]
Let me test that and try to reproduce it

**Pull request merged**